.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,189 @@
+        - ARM v7
+        - ARM64
+    validations:
+      required: true
+  - type: dropdown
+    id: container-orchestrator
+    attributes:
+      label: What container orchestration tool are you using?
+      options:
+        - Docker
+        - Docker Compose
+        - Podman
+        - Kubernetes (or something similar)
+        - Something else
+    validations:
+      required: true
+  - type: input
+    id: docker-version
+    attributes:
+      label: Docker version
+      description: Please show us the output of `docker --version` on the host your mail server is running on if you are using Docker. Note that if you are not using Docker, there is no official support provided by maintainers as DMS mainly focuses on Docker (and Docker Compose).
+      placeholder: Docker version 20.10.7, build f0df350
+    validations:
+      required: false
+  - type: input
+    id: docker-compose-version
+    attributes:
+      label: Docker compose version
+      description: Please show us the output of `docker-compose --version` on the host your mail server is running on. Note that if you are not using Docker Compose, there is no official support provided by maintainers as DMS mainly focuses on Docker COmpose (and Docker).
+      placeholder: docker-compose version 1.28.6, build 5db8d86f
+    validations:
+      required: false
+  - type: input
+    id: uname-a
+    attributes:
+      label: The output of `uname -a`
+      description: To double-check that the information above is correct, please paste the output of `uname -a` here.
+      placeholder: Linux swift 5.11.0-31-generic \#33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+    validations:
+      required: true
+  - type: textarea
+    id: important-environment-variables
+    attributes:
+      label: Important environment variables
+      description: If there are environment variables important to this issue, please post them here. If we ask for the values of variables, make sure to put them in here too.
+      render: bash
+  - type: textarea
+    id: relevant-log-output
+    attributes:
+      label: Relevant log output
+      description: Show us relevant log output here. You can enable debug output by setting `DMS_DEBUG` to `1`.
+      render: shell
+  - type: textarea
+    id: other-relevant-information
+    attributes:
+      label: Other relevant information
+      description: If there is more, you can tell us here. Make sure to use Markdown formatting here.
+      render: Markdown
+  - type: checkboxes
+    id: terms-code-of-conduct
+    attributes:
+      label: Code of conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md).
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+        - label: I have read this project's Code of Conduct
+          required: true
```

Comment:
Imho this is unnecessarily doubled. We do not want to close a deal with someone

Reply:
I see where you're coming from. And I guess adding a checkbox won't help the situation much.

Reply:
Done. [ba9ea5a85f812e811782bb27864d87ed2916c056]